src/globus_sdk/scopes/data/timer.py

```diff
@@ -0,0 +1,12 @@
+from ..builder import ScopeBuilder
+
+TimerScopes = ScopeBuilder(
+    "524230d7-ea86-4a52-8312-86065a9e0417",
+    known_url_scopes=[
+        "timer",
+    ],
+)
+"""Globus Timer scopes.
```

Comment:
Although this should likely be named "TimersScopes", the name of the service should be updated.

```suggestion
"""Globus Timers scopes.
```